doc/content/integrations/influxdb-cloud/telegraf-setup/_index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Telegraf Setup"
+description: ""
+weight: 1
+---
+
+This section contains instructions to configure the Telegraf agent for connecting to {{% tts %}} MQTT server and sending data to InfluxDB Cloud 2.0.
+
+<!--more-->
+
+The information needed to configure Telegraf can be found on the **MQTT** tab in {{% tts %}} **Integrations** menu.
+
+{{< figure src="tts-mqtt-info.png" alt="The Things Stack MQTT server info" >}}
+
+Update the previously downloaded Telegraf configuration with the following lines and modify them according to your MQTT server info:
+
+```bash
+[[inputs.mqtt_consumer]]
+#
+# MQTT broker URLs to be used. The format is scheme://host:port, schema can be tcp, ssl, or ws.
+  servers = ["tcp://127.0.0.1:1883"]
```

Comment:
Shouldn't the schema be `mqtt:` or `mqtts:` ? @adriansmares

Reply:
Our pub/sub driver requires `mqtt` or `mqtts`, but the one Telegraf uses accepts `tcp`. You can see this in the comment above the line.

Reply:
ah but they all resolve to the same type of tls request?

Reply:
Yes, fundamentally `tls`, `ssl`, `tcps` and `mqtts` are equivalent for our pub/sub driver, while the one Telegraf uses accepts only `tls` and `ssl` (but they are still equivalent).

SSL is the precursor to TLS, and while technologically speaking they are different, the terms are used interchangeability nowadays and they both refer to TLS. I personally have yet to meet a program/library that interprets `tls://` and `ssl://` differently.

You may say that there are a lot of terms that actually mean the same thing and this leads to confusion, I couldn't agree more, but this is the way TLS and SSL are now surviving in our lingo.